### ISSUE TITLE
YOLO26-Depth

### ISSUE THREAD

**Issue by ycy-111-666** [ultralytics/ultralytics#25474]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

It's great to see how rapidly YOLO has developed. The newly launched monocular depth estimation feature is truly impressive. However, when I used the currently released pre-trained model for inference, I found that the results were not as good as those shown on the release page (the arm of the person on the right side was not detected). The current inference result is based on the model yolo26x-depth.pt. Why is this the case?
<img width="986" height="724" alt="Image" src="https://github.com/user-attachments/assets/392652dc-09bc-4dc7-812a-9da9c4c00cc5" />
<img width="410" height="574" alt="Image" src="https://github.com/user-attachments/assets/6dfb3eea-2cb5-42ac-9581-b86cc5bdc12e" />

**Comment by UltralyticsAssistant** [comment]
👋 Hello @ycy-111-666, thank you for your interest in Ultralytics and YOLO26 🚀! This is an automated response, and an Ultralytics engineer will also assist you soon.

We recommend visiting the [Docs](https://docs.ultralytics.com/) for new users, including [Python](https://docs.ultralytics.com/usage/python) and [CLI](https://docs.ultralytics.com/usage/cli) usage examples and answers to common questions.

To help us investigate the difference between your `yolo26x-depth.pt` results and the release-page example, please provide:
- The exact Ultralytics version and installation command
- Your operating system, Python version, PyTorch version, and GPU/CPU details
- The complete inference command or a small Python script
- The original input image or a comparable image, if permitted
- Any inference arguments or preprocessing changes
- The complete console output and generated result

If this is a 🐛 Bug Report, please also provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example) so the behavior can be reproduced and debugged.

For custom training ❓ questions, please provide as much information as possible, including dataset image examples and training logs, and verify that you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments, with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes) and [Tasks](https://docs.ultralytics.com/tasks) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
@Bovey0809 Can you check?

**Comment by Y-T-G** [comment]
@ycy-111-666 Hello. This result is using `imgsz=1024`. You can try using `imgsz=1024` to improve the quality of depth map

**Comment by glenn-jocher** [comment]
You can try different sizes at https://platform.ultralytics.com/ultralytics/yolo26/yolo26x-depth?tab=predict

<img width="1476" height="1674" alt="Image" src="https://github.com/user-attachments/assets/d3c6fc45-2fe5-4280-bb91-3e741287529d" />

**Comment by ycy-111-666** [comment]
@glenn-jocher @Y-T-G Nice, i get it. Excellent result. Thank you very much.

<img width="810" height="1026" alt="Image" src="https://github.com/user-attachments/assets/eeb70d98-4aef-40ae-a37a-67e20cd10500" />
